src/client/services/les-entreprises-s-engagent/lesEntreprisesSEngagent.service.ts

```diff
@@ -0,0 +1,12 @@
+import { HttpClientService } from '~/client/services/httpClient.service';
+import { FormulaireEngagement } from '~/pages/les-entreprises-s-engagent/inscription';
+import { createSuccess, Either } from '~/server/errors/either';
+
+export class LesEntreprisesSEngagentService {
+  constructor(private httpClientService: HttpClientService) {}
+
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  async envoyerFormulaireEngagement(formulaire: FormulaireEngagement): Promise<Either<void>> {
+    return createSuccess(undefined);
```

Comment:
si c'est pas encore implémenté, c'est possible de retourner un Failure plutôt ? :D ou throw Error('not implemented') ?

Reply:
Aaah j'ai implémenter le message quand tout c'est bien passer c'est pour ca que j'ai mis le success, ca me semblait plus judicieux